### PR TITLE
Fix- when saving offline, instead of project name the object was showing

### LINKED ISSUE #2000: Incorrect project name while saving the project for offline
**Describe the bug**
Incorrect project name while saving the project for offline.
> It shows `[object HTMLSpanElement]` instead of project name

**To Reproduce**
1. Go to Simulator
2. Click on save for offline
3. See error

**Expected behavior**
It should show the project name instead of `[object HTMLSpanElement]`

**Screenshots**
image

**Device Information [optional]:**
 - OS: Windows 10 20H2
 - Browser Chrome
 - Version 86

**Additional context**
Probably error in parsing HTML element to text

**Are you working on this issue? (Yes/No)**
No

### PR DESCRIPTION
**Pull request by dipanshparmar**
Fixes #2000

### Screenshots of the changes (If any) -
**Before**
Screenshot from 2020-12-05 19-24-42

**After**
Screenshot from 2020-12-05 19-26-33

### CODE REVIEW

File: public/src/data/project.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ export function saveOffline() {
     const temp = JSON.parse(localStorage.getItem('projectList')) || {};
     temp[projectId] = getProjectName();
     localStorage.setItem('projectList', JSON.stringify(temp));
-    showMessage(`We have saved your project: ${projectName} in your browser's localStorage`);
+    showMessage(`We have saved your project: ${projectName.textContent} in your browser's localStorage`);
```
Comment (satu0king) [inline public/src/data/project.js:77]: Just use `getProjectName()`

**Comment by satu0king** [comment]
@dipanshparmar can u check #2026

File: public/src/data/project.js (hunk 1 above)
Comment (dipanshparmar) [reply public/src/data/project.js]: Done @satu0king

**Comment by dipanshparmar** [comment]
> @dipanshparmar can u check #2026

Sure @satu0king

**Pull request closed without merging**